README.md

```diff
@@ -63,13 +72,13 @@ Running Compliance Checker on the dataset from: compliance_checker/tests/data/ru
 
 
 --------------------------------------------------------------------------------
-                    The dataset scored 100 out of 188 points                    
-                             during the acdd check                              
```

Comment:
There are a few blank spaces removal due to my editor setting. Let me know if that is a nuisance and I will edit the PR.